pkgs/os-specific/linux/lm-sensors/default.nix

```diff
@@ -19,8 +19,8 @@ stdenv.mkDerivation rec {
    ++ lib.optional sensord rrdtool;
 
   makeFlags = [
-    "PREFIX=${placeholder "out"}"
-    "ETCDIR=${placeholder "out"}/etc"
+    "DESTDIR=${placeholder "out"}"
```

Comment:
`DESTDIR` is problematic https://github.com/NixOS/nixpkgs/issues/65718. Try overriding `ETCDIR` in `installFlags`.

Reply:
@jtojnar Thank you for your suggestion. I have removed `DESTDIR` from `makeFlags` and moved ETCDIR into `installFlags`.